### ISSUE TITLE
[resources] Link to rss.com rather than embedding spotify

### ISSUE THREAD

**Issue by Atmosfearful** [KlimaDAO/klimadao#908]
klimadao.finance/resources currently shows embedded spotify iframes for podcast cards.
Instead, let's hyperlink out to rss.com

# Why
- better for people who don't use spotify
- rss shows a popup that lets you pick where to listen (apple, spotify, rss.com, amazon, google)
- most people probably don't listen right in the browser (fragile if you accidentally navigate elsewhere, close the tab etc.)
- better performance, not fetching a ton of podcast data
- fix #890 safari bug

# Requirements
- CMS `podcast` schema will need to be enhanced so authors can enter an rss.com id
- this id is used to construct a url like this: [https://rss.com/podcasts/potk/743034/?listen-on=true](https://rss.com/podcasts/potk/743034/?listen-on=true)
  - `listen-on=true` means that the user is prompted to choose their favorite listening app, or close the modal and listen on rss.com
- the design needs to be updated before dev can start @theprofessora 

# Implementation
- 1st PR: add the above rss properties to the sanity schema. Atmos will deploy the schema updates to the sanity studio ui.
- once it is deployed to the sanity studio, update all the `podcast` documents in the cms to contain the new rss.com id
- 2nd PR: swap out the spotify embed with the rss.com link and the new card design
- delete the old spotify embed codes from all the podcast documents in the cms
- 3rd PR: update the cms again to remove the spotify entry from the studio

**Comment by Atmosfearful** [comment]
I am waiting on final confirmation from the POTK team and for a design before we can start on this.

In the mean time @0xMakka you can get familiar with the `cms` folder and the basics of sanity schemas https://www.sanity.io/docs/schema-types

**Comment by 0xMakka** [comment]
Thanks @theprofessora and @Atmosfearful, will pick this up later today.
